### ISSUE TITLE
Accessibility view does not contain a command that agent is asking to run

### ISSUE THREAD

**Issue by mo29cg** [microsoft/vscode#246302]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: Version: 1.99.0
- OS Version: Windows 11 Pro

Steps to Reproduce:

1. Ask copilot agent to do task including running cli commands.
2. select agent message, then press Alt + F2

then the opened view does not contain the command that agent is asking for a permission.

**Comment by mo29cg** [comment]
> We include the user action required info in the aria label of the chat response - do you not feel that is sufficient here?

The command it's asking to run is not read out in either when scrolling the chat history with up/down arrows or in the accessibility view.
The only way to see the command right now is switching to "browse mode" of NVDA, but it makes VSCode buggy, it's not really an option.

**Comment by meganrogge** [comment]
Using VoiceOver on mac, when an action is required, I hear: Action required: Run command in terminal. If I miss that, and use ctrlCmd+ up arrow to focus the response, I hear that again, Action required: Run command in terminal. 

I agree this can also be added to the accessible view.

**Comment by mo29cg** [comment]
> Using VoiceOver on mac, when an action is required, I hear: Action required: Run command in terminal. If I miss that, and use ctrlCmd+ up arrow to focus the response, I hear that again, Action required: Run command in terminal.

Does that mean VoiceOver announces the command itself like "cd server/ && pnpm run"?
NVDA does not announce that.

**Comment by meganrogge** [comment]
It does not announce the particular command. I figured that would be too verbose - @jooyoungseo, thoughts? We could add that to both the aria label and the alert.

**Comment by dennischenfeng** [comment]
+1. FWIW, here are my thoughts as an NVDA user. Apologies if I err on the side of verbose.
I'd prefer the actual text of the command (like "pip install numpy") to at least be in the Accessible View (Alt+F2).
As for whether they be in aria label (only spoken in Browse mode as mentioned above which is clunky and I don't love switching to in VS code) or whether they are in the audio alert, I don't have a strong opinion.

I rely on the Accessible View the most to actually read through paragraphs of text, rather than browse mode or alerts.
My way of thinking about the Accessible View is that ideally it should have all the text there that a sighted person could see, which includes the CLI commands.

**Comment by mo29cg** [comment]
> It does not announce the particular command. I figured that would be too verbose - [@jooyoungseo](https://github.com/jooyoungseo), thoughts? We could add that to both the aria label and the alert.


Does VoiceOver let you move the focus to the proposed command section, that's how you see the command?
This is not really doable with NVDA. 
I don't think VSCode accessibility is made to comply with browse mode of NVDA, browse mode make it weird like when I switch to browse mode then go back to focus mode, the cursor often slip off my editor when I'm reading a file.

**Comment by mo29cg** [comment]
https://code.visualstudio.com/docs/configure/accessibility/accessibility#_screen-readers

>For NVDA, we recommend that you stay in focus mode and use the hotkeys to navigate, instead of using browse mode.

The documentation states we should not switch to browse mode, but the command is only available on browse mode, this is not good.

**Comment by mo29cg** [comment]
Thank you for the quick fix!

**Comment by meganrogge** [comment]
Steps:
- invoke a tool request via `run command ls in terminal` or `run task X`
- verify that when in screen reader mode, with focus in the chat input or on the chat response, you can review the command that will run in the accessible view via `alt+f2`
